src/js.rs

```diff
@@ -590,6 +571,44 @@ extern "C" {
     pub fn to_string(this: &Error) -> JsString;
 }
 
+// Float32Array
+#[wasm_bindgen]
+extern "C" {
+    pub type Float32Array;
+
+    /// The `Float32Array()` constructor creates an array of 32-bit floats.
+    ///
+    /// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Float32Array
+    #[wasm_bindgen(constructor)]
+    pub fn new(constructor_arg: JsValue) -> Float32Array;
+
+    /// The fill() method fills all the elements of an array from a start index
+    /// to an end index with a static value. The end index is not included.
+    ///
+    /// http://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/fill
+    #[wasm_bindgen(method)]
+    pub fn fill(this: &Float32Array, value: JsValue, start: u32, end: u32) -> Float32Array;
```

Comment:
This was previously `JsValue` here but I wonder if this would be better bound as `f32`? (or the appropriate type on each array below)

Reply:
Seems sensible to me.  Perhaps the `JsValue` was there only because that's the type on `Array.fill()`